src/tasks/Microsoft.NET.WebAssembly.Webcil/Microsoft.NET.WebAssembly.Webcil.csproj

```diff
@@ -17,8 +17,8 @@
   <ItemGroup>
     <!-- we need to keep the version of System.Reflection.Metadata in sync with dotnet/msbuild and dotnet/sdk -->
     <PackageReference Include="System.Memory" Version="$(SystemMemoryVersion)" /> 
-    <PackageReference Include="System.Reflection.Metadata" Version="$(SystemReflectionMetadataVersion)" />
-    <PackageReference Include="System.Collections.Immutable" Version="$(SystemCollectionsImmutableVersion)" />
+    <PackageReference Include="System.Reflection.Metadata" Version="$(SystemReflectionMetadataToolsetVersion)" />
+    <PackageReference Include="System.Collections.Immutable" Version="$(SystemCollectionsImmutableToolsetVersion)" />
```

Comment:
we probably need the same changes in MonoTargetsTasks.csproj and WasmAppBuilder.csproj

Reply:
hm but if I see it correctly then nobody uses the SystemReflectionMetadataVersion anymore so we probably don't need to reference the current version?

Reply:
MonoAOTCompiler.csproj, installer.tasks.csproj and ILLink.Tasks.csproj also use `SystemReflectionMetadataVersion` but those should also use the toolset version instead since they're msbuild tasks

Reply:
Do the task projects need explicit reference? If by removing them the dependencies get resolved transitively, I think we should do that? Assuming these tasks were used in the CI, it seems to work. But I agree we can do better. If we can't test the "live" version that would explain why they didn't fail.